internal/resource-request-operator/resourceRequest_controller_test.go

```diff
@@ -27,8 +30,39 @@ var _ = Describe("ResourceRequest controller", func() {
 
 	Context("Testing ResourceRequest Controller when creating a new ResourceRequest", func() {
 		It("Should create new ResourceRequest and related ResourceOffer ", func() {
-			By("By creating a new ResourceRequest")
 			ctx := context.Background()
+			By("Creating mock nodes")
+			resources := corev1.ResourceList{}
+			resources[corev1.ResourceCPU] = *resource.NewQuantity(2, resource.DecimalSI)
+			resources[corev1.ResourceMemory] = *resource.NewQuantity(1, resource.BinarySI)
+			resources[corev1.ResourceLimitsCPU] = *resource.NewQuantity(3, resource.DecimalSI)
+			node1 := &corev1.Node{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-node1",
+					Namespace: "default",
```

Comment:
The nodes are cluster scoped resources, they do not require setting the namespace